mdm/common/src/main/java/com/gentics/mesh/core/data/dao/PersistingLanguageDao.java

```diff
@@ -12,13 +13,20 @@
  * @author plyhun
  *
  */
-public interface PersistingLanguageDao extends LanguageDao, PersistingDaoGlobal<HibLanguage> {
+public interface PersistingLanguageDao extends LanguageDao, PersistingDaoGlobal<HibLanguage>, PersistingNamedEntityDao<HibLanguage> {
 
 	default HibLanguage create(String languageName, String languageTag, String uuid) {
 		HibLanguage language = createPersisted(uuid);
 		language.setName(languageName);
 		language.setLanguageTag(languageTag);
-		return mergeIntoPersisted(language);
+		mergeIntoPersisted(language);
+
+		maybeGetCache().ifPresent(cache -> {
+			cache.clear(languageName);
+			cache.clear(languageTag);
+			CommonTx.get().data().mesh().batchProvider().get().add(language.onCreated());
```

Comment:
Here and below - we clear the cache explicitly, and sending the creation event to notify the cluster to clear it as well.